packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -18,11 +18,17 @@ type Identifier = Pick<Patient, "cxId" | "externalId"> & { facilityId: string };
 type PatientNoExternalData = Omit<PatientData, "externalData">;
 export type PatientCreateCmd = PatientNoExternalData & Identifier;
 
-export const createPatient = async (
-  patient: PatientCreateCmd,
-  forceCommonwell?: boolean,
-  forceCarequality?: boolean
-): Promise<Patient> => {
+export const createPatient = async ({
+  patient,
+  rerunPdOnNewDemographics = false,
```

Comment:
Can we also expose the `augmentDemographics` here and on the endpoint? It shouldn't impact cxs as its optional, its not yet documented, but we could easily offer to certain cxs w/o change in code.

Reply:
For create, in theory there are no existing demographics. It's added to update.

Reply:
Yes. I will explain tomorrow but the original idea behind two query params was that the second one tells the code to use augmented data *for the initial request too* -- allowing us to create augmentations off of augmentations :). But again, if we think one level is enough for now, one query param will work.

Reply:
Irrelevant now: flow has been updated to de-couple rerun and scheduling. Went down to one query param as we now always use augmented data per our discussion.